Detectors/MUON/MID/Conditions/src/DCSNamer.cxx

```diff
@@ -0,0 +1,126 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MIDConditions/DCSNamer.h"
+
+#include <array>
+#include <fmt/printf.h>
+#include <iostream>
+#include <set>
+#include <optional>
+#include <fmt/format.h>
+
+namespace
+{
+const std::string detPrefix = "MID";
+
+// clang-format off
+std::array<int, 72> detElemIds = {
```

Comment:
Notice that the DetElemId changed in Run3. They now go from 0 to 72. So this is actually obsolete.
In particular:
This is INSIDE:
0 -> 1114; 1 -> 1115; 2->1116; 3->1117; 4->1100; 5->1101; 6->1102; 7->1103; 8->1104;
9->1214; 10->1215...
18->1314; 19->1315...
27->1414; 28->1415..

And this is OUTSIDE:
36->1113; 37->1112; 38->1111; 39->1110; 40->1109; 41->1108; 42->1107; 43->1106; 44->1105
45->1213; 46->1212...
54->1313; 46->1312...
63->1413; 46->1412...

Reply:
And all of the needed transformations can be found in MIDBase/DetectorParameters.h

Reply:
ah, ok, indeed I've completely missed the transition for 11xx to 0..71 ! Will change that. Thanks.